pandas/core/generic.py

```diff
@@ -1940,6 +1940,79 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n=None, frac=None , replace=False, weights=None, seed=None):
+        """
+        Returns a random sample of rows from object. 
+        
+        Parameters
+        ----------
+        n : int, optional
+            Number of rows to return. Cannot be used with frac.
+            Default = 5 if frac = None. 
+        frac : float 
+            Share of rows to return. Cannot be used with n. 
+        replace : boolean
+            Sample with or without replacement. Default = False. 
```

Comment:
I usually let function signatures document default values like `False` instead of repeating it in the docstring, but that's not a strong rule.